lib/go-tc/traffic_stats.go

```diff
@@ -270,3 +270,24 @@ func MessagesToString(msgs []influx.Message) string {
 	b.WriteRune(']')
 	return b.String()
 }
+
+// TrafficStatsCDNStats contains summary statistics for a given CDN
+type TrafficStatsCDNStats struct {
+	Bandwidth    *float64 `json:"bandwidth"`
+	Capacity     *float64 `json:"capacity"`
+	CDN          string   `json:"cdn"`
+	Connnections *float64 `json:"connections"`
+}
+
+// TrafficStatsTotalStats contains summary statistics across CDNs
+// Different then TrafficStatsCDNStats as it omits Capacity
+type TrafficStatsTotalStats struct {
+	Bandwidth    *float64 `json:"bandwidth"`
+	CDN          string   `json:"cdn"`
+	Connnections *float64 `json:"connections"`
+}
+
+// TrafficStatsCDNStatsResponse contains response for getting current stats
+type TrafficStatsCDNStatsResponse struct {
+	Response []TrafficStatsCDNStats `json:"response"`
```

Comment:
The Perl handler returned everything in `response.currentStats`, not just in `response`. Frankly I think what you have is better, but it is a breaking change.

example Perl response:
```http
HTTP/1.1 200 OK
Connection: keep-alive
Access-Control-Allow-Credentials: true
Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept
Set-Cookie: mojolicious=...; expires=Fri, 22 Nov 2019 21:15:49 GMT; path=/; HttpOnly
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Date: Fri, 22 Nov 2019 17:15:49 GMT
Access-Control-Allow-Origin: *
Content-Encoding: gzip
Vary: Accept-Encoding
Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
Content-Length: 148
Content-Type: application/json
Server: Mojolicious (Perl)

{
	"response": {
		"currentStats": [
			{
				"bandwidth": null,
				"cdn": "ALL",
				"connections": null,
				"capacity": null
			},
			{
				"bandwidth": 0.0001154,
				"cdn": "CDN-in-a-Box",
				"connections": 4,
				"capacity": 17
			},
			{
				"bandwidth": 0.0001154,
				"cdn": "total",
				"connections": 4
			}
		]
	}
}
```

Reply:
good catch let me fix this

Reply:
fixed